v3/src/components/axis/hooks/use-sub-axis.ts

```diff
@@ -214,7 +214,7 @@ export const useSubAxis = ({
             )
         }
 
-      d3Scale.range(axisIsVertical ? [rangeMax, rangeMin] : [rangeMin, rangeMax])
+      d3Scale.range(newRange)
```

Comment:
We set the range on this scale when we created it in line 77. Seems like it shouldn't be necessary to set it again here.